packages/block-library/src/social-link/index.js

```diff
@@ -7,42 +7,30 @@ import { __ } from '@wordpress/i18n';
  * Internal dependencies
  */
 import edit from './edit';
-import socialList from './social-list';
+import patterns from './patterns';
 
-const commonAttributes = {
+export const name = 'core/social-link';
+
+export const settings = {
+	title: __( 'Social Icon' ),
 	category: 'widgets',
 	parent: [ 'core/social-links' ],
 	supports: {
 		reusable: false,
 		html: false,
 	},
 	edit,
+	description: __( 'Link to a social media profile' ),
```

Comment:
I'm not very happy with this description, as the patterns offer many kinds of Web-based resources: social media profiles, multimedia content, misc. pages.